src/lib/list/selection-list.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  AfterContentInit,
+  Component,
+  ContentChildren,
+  ElementRef,
+  Input,
+  Optional,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  OnDestroy,
+  EventEmitter
+} from '@angular/core';
+import {coerceBooleanProperty, SelectionModel} from '../core';
+import {FocusKeyManager} from '../core/a11y/focus-key-manager';
+import {Subscription} from 'rxjs/Subscription';
+import {SPACE} from '../core/keyboard/keycodes';
+import {Focusable} from '../core/a11y/focus-key-manager';
+import {MdListOption} from './list-option';
+import {CanDisable, mixinDisabled} from '../core/common-behaviors/disabled';
+import 'rxjs/add/observable/merge';
+import 'rxjs/add/observable/fromEvent';
+import 'rxjs/add/operator/filter';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/observable/of';
+import {merge} from 'rxjs/operator/merge';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/startWith';
```

Comment:
:+1:  Changed to use '`RxChain`' instead of directly importing `'observer/add/`'Changed to use 'RxChain' instead of directly importing 'observer/add/'